src/test/java/com/acrolinx/proxy/TomcatIT.java

```diff
@@ -103,7 +103,7 @@ private static void verifyInteractionWithWireMock(WireMockServer wireMockServer)
   }
 
   @Test
-  void embeddedTomcatTest(@TempDir Path tempDirectory) throws Exception {
+  void integrationTest(@TempDir Path tempDirectory) throws Exception {
```

Comment:
Why was this renamed? I find integrationTest a bit too generic.

Reply:
nit: I think embeddedTomcatIntegrationTest would be better

Reply:
I just renamed the method to embeddedTomcatIntegrationTest, thanks @willynganga for the suggestion.